src/configuration.ts

```diff
@@ -15,15 +15,23 @@ export default class Config {
   static models: Array<typeof Model> = [];
   static typeMapping: Object = {};
   static logger: Logger = new Logger();
+  static jwtLocalStorage: string | boolean = 'jwt';
```

Comment:
I think more correct is `string | false` since a value of true would throw an error when using it to set the value.

Reply:
Also while it makes sense to default on, there is no localStorage API on the server, so we should probably have it default more intelligently depending on whether the API exists.

Reply:
`Config.jwtLocalStorage` would `== 'jwt'`, but [no localStorage logic would fire](https://github.com/richmolj/jsorm-1/blob/master/src/util/refresh-jwt.ts#L8). So I think this is OK.

Good catch on `false`.